pydantic/functional_validators.py

```diff
@@ -162,19 +186,36 @@ def __get_pydantic_core_schema__(self, source_type: Any, handler: _GetCoreSchema
 
         try:
             schema = handler(source_type)
-            serialization = core_schema.wrap_serializer_function_ser_schema(function=lambda v, h: h(v), schema=schema)
+            serialization = core_schema.wrap_serializer_function_ser_schema(
+                function=lambda v, h: h(v),
+                schema=schema,
+                return_schema=schema,
+            )
```

Comment:
Note: the added `return_schema` allows for the following to not crash (it did before):

```python
from typing import Annotated

from pydantic import PlainValidator, BaseModel


class MyModel(BaseModel):
    a: Annotated[int, PlainValidator(lambda x: x)]


print(MyModel.model_json_schema(mode='serialization'))
```

Reply:
Can we do this in a different PR, just to make the diff smaller / more streamlined here?